spec/overlap/multi_part_overlap_spec.rb

```diff
@@ -20,7 +27,6 @@
     build(:ht_item,
       :mpm,
       enum_chron: "",
-      n_enum: "",
```

Comment:
n_enum is a computed field; setting it here directly caused some unexpected issues.